setup.cfg

```diff
@@ -59,6 +59,11 @@ full =
     statsmodels
     timezonefinder
 
+[flake8]
+max-line-length = 88
+select = C,E,F,W
+extend-ignore = E203,W503
```

Comment:
I think the issues raised by flake8 are generally good and useful. I'd be happy if someone sat down, worked through them and submitted a later "fix flake8 issues" PR that made flake8 not fail. It's probably best for one person to just do something sensible rather than discussing each item one by one.

The only flake8 issue which sticks out to me as problematic is "E731 do not assign a lambda expression, use a def". When setting up a constraint for an Iris cube, it's idiomatic to use a lambda for more complex constraints. For example, see the orange warning box at https://scitools-iris.readthedocs.io/en/stable/userguide/subsetting_a_cube.html#cube-extraction
I'd suggest adding E731 to the extend-ignore config line.

Reply:
Agreed on E731, happy to ignore that one. I'll bag the fix-flake8-issues if that's OK.

Reply:
> submitted a later "fix flake8 issues" PR

Sounds good to me.

> I'd suggest adding E731 to the extend-ignore config line.

E731 refers to assigning a lambda to a variable. The example linked to passes a lambda in as a function argument which E731 is fine with. Having said that, I'm happy to ignore E731.

> I'll bag the fix-flake8-issues if that's OK.

Thanks @benfitzpatrick.